ee_plugin/ee_plugin.py

```diff
@@ -112,36 +112,54 @@ def initGui(self):
             parent=self.iface.mainWindow(),
             triggered=self._run_cmd_set_cloud_project,
         )
+        add_fc_button = QtWidgets.QAction(
+            text=self.tr("Add Feature Collection"),
+            parent=self.iface.mainWindow(),
+            triggered=lambda: forms.add_feature_collection_form(self.iface),
+        )
 
-        # Build plugin menu
+        # Initialize plugin menu
         plugin_menu = cast(QtWidgets.QMenu, self.iface.pluginMenu())
-        ee_menu = plugin_menu.addMenu(
+        self.menu = plugin_menu.addMenu(
             icon("earth-engine.svg"),
             self.tr("&Google Earth Engine"),
         )
-        self.menu = ee_menu
-        ee_menu.addAction(ee_user_guide_action)
-        ee_menu.addSeparator()
-        ee_menu.addAction(sign_in_action)
-        ee_menu.addAction(self.set_cloud_project_action)
-
-        # Build toolbar
-        toolButton = QtWidgets.QToolButton()
-        toolButton.setToolButtonStyle(
+
+        # Initialize toolbar menu
+        self.toolButton = QtWidgets.QToolButton()
+        self.toolButton.setToolButtonStyle(
             Qt.ToolButtonStyle.ToolButtonIconOnly
             # Qt.ToolButtonStyle.ToolButtonTextBesideIcon
         )
-        toolButton.setPopupMode(
-            QtWidgets.QToolButton.ToolButtonPopupMode.MenuButtonPopup
+        self.toolButton.setPopupMode(
+            # QtWidgets.QToolButton.ToolButtonPopupMode.DelayedPopup  # Button is only for triggering action
```

Comment:
Should we remove these commented out buttons?

Reply:
Yeah, probably.  I did some playing around to understand what each PopupMode represented and commented that.  I left that comment to save others from having to do that research, but I'm not really sure that adds any real value to anyone.

Reply:
Closed via [6b2a2f9](https://github.com/gee-community/qgis-earthengine-plugin/pull/231/commits/6b2a2f9f7b8ac4d0e9ed51f9f29974bb7397ae51)